src/worker/dump_db.rs

```diff
@@ -246,6 +253,22 @@ impl Drop for DumpTarball {
     }
 }
 
+struct InvalidateCaches {}
+
+impl InvalidateCaches {
+    pub fn create(client: &Client, target_name: &str) -> Result<(), PerformError> {
```

Comment:
any reason for this empty struct? maybe `fn invalidate_caches()` would be simpler? :)